crates/ee-chain-ssz-types/src/block.rs

```diff
@@ -0,0 +1,100 @@
+// SSZ type definitions for execution environment block structures.
+// Types defined here match the pythonic schema in `schemas/ee-chain-types.ssz`.
+
+use ssz_derive::{Decode, Encode};
+use ssz_types::VariableList;
+use strata_acct_types::{AccountId, BitcoinAmount, Hash, SentMessage, SubjectId};
+use tree_hash_derive::TreeHash as TreeHashDerive;
+
+// Constants for list bounds (from schema)
+pub const MAX_SUBJECT_DEPOSITS_PER_BLOCK: usize = 1 << 16; // 65536
+pub const MAX_OUTPUT_TRANSFERS_PER_BLOCK: usize = 1 << 16; // 65536
+pub const MAX_OUTPUT_MESSAGES_PER_BLOCK: usize = 1 << 16; // 65536
+
+/// Variable-length list for subject deposits
+type SubjectDepositList = VariableList<SubjectDepositData, MAX_SUBJECT_DEPOSITS_PER_BLOCK>;
+
+/// Variable-length list for output transfers
+type OutputTransferList = VariableList<OutputTransfer, MAX_OUTPUT_TRANSFERS_PER_BLOCK>;
+
+/// Variable-length list for output messages
+type OutputMessageList = VariableList<SentMessage, MAX_OUTPUT_MESSAGES_PER_BLOCK>;
+
+/// Container for an execution block that signals additional data with it.
+// TODO better name, using an intentionally bad one for now
+/// Schema: class ExecBlockNotpackage(Container)
```

Comment:
I think this is broken.
I don't know if doing comments between docstrings is supported.
Move the TODO 1 line below after the docstrings.